app/assets/javascripts/discourse/app/widgets/sidebar-toggle.js

```diff
@@ -0,0 +1,16 @@
+import { createWidget } from "discourse/widgets/widget";
```

Comment:
Would it be hard to add this toggle as a Glimmer component?

Reply:
Since this is going to be rendered on page load, I think it'll be simpler to keep this as a widget for now until we completely drop it.